lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -612,5 +612,162 @@ def test_laea_cs(self):
         self._test(coord_system, expected)
 
 
+@tests.skip_data
+class Test_packed_data(tests.IrisTest):
+    def _get_scale_factor_add_offset(self, cube, datatype):
```

Comment:
@jswanljung If you agree to split this test class (as mentioned below) then this `method` is really a convenience `function` that would live at the module level (note no usage of `self` within the method) ... then each of the test cases can still call it as a module level `function`.